### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by zees-dev** [johncintron/nodin#8]
Hey is this project dead or inactive?

I think this project has great potential and would be keen to start working on it - but would need some guidance.
Anyone doing any work on JavaScript based MapleStory?

**Comment by 100tomer** [comment]
I agree with you, completing this project can be amazing. Did you success in running it and pass the login screen?

**Comment by Zontex** [comment]
@zees-dev @100tomer I created a new fork to continue the progress, already made some progress: https://github.com/Zontex/nodin-maplestroy-js-v62/ completing this project will be almost impossible... if we account for the quests and everything. tons of work. but a good progress can be made to make old-school experience...
